dagmc_stats.py

```diff
@@ -30,16 +28,14 @@ def get_dagmc_tags(my_core):
     return dagmc_tags
 
 
-def get_entity_ranges(my_core, meshset, entity_types):
+def get_entity_ranges(my_core, meshset, entity_types, dagmc_tags):
```

Comment:
Suggestion: Add default value for parameter `dagmc_tags` for better compatibility.

Reply:
The default value that would be put there is generated from the previous function; how could I transfer that from the other function to be a parameter of this function?

Reply:
The `dagmc_tags` are used when and only when `entity_type == 11`. Therefore, it is an optional parameter. For most cases, you don't need to pass in this parameter when you call this function. Add a default value could make it cleaner.

You can just set it to be `None`. And check the value before you use it:

Reply:
Thanks, will implement this